### DISCUSSION TITLE
Floret vectors for Italian

### DISCUSSION THREAD

**Discussion by darioprencipe** [explosion/spaCy#13094] [Language Support]
Hello team,

not sure whether `spaCy` is the right forum for that but - do you consider producing `floret` vectors for Italian language any time soon? If not, do you know someone who already worked on that beforehanb?

Thanks a lot!

**Comment by adrianeboyd** [comment]
We don't have any concrete plans to publish floret vectors for Italian, but we've created a [spacy project](https://spacy.io/usage/projects) that shows how to train floret vectors:

https://github.com/explosion/projects/tree/v3/pipelines/floret_wiki_oscar_vectors

```shell
python -m spacy project clone pipelines/floret_wiki_oscar_vectors
```

You would need a server with a large amount of hard drive space and a good CPU.

--------

We also have a linux-only project with slightly more complicated requirements, which is what we use internally to train vectors for the published trained pipelines with floret vectors (Korean, Swedish, etc.):

https://github.com/explosion/spacy-vectors-builder

This project has several language-specific branches that include the settings chosen for each language depending on the amount of training data available and the morphology and writing system.
